components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/listener/IdentityOathEventListener.java

```diff
@@ -352,6 +354,11 @@ private void removeAccessTokensFromCache(Set<AccessTokenDO> accessTokenDOSet) {
 
         if (CollectionUtils.isNotEmpty(accessTokenDOSet)) {
             for (AccessTokenDO accessTokenDO : accessTokenDOSet) {
+                if (StringUtils.isNotEmpty(accessTokenDO.getGrantType()) &&
```

Comment:
StringUtils.equalsIgnoreCase is null safe.

Reply:
Fixed with 2947e59cfc6e711d49cbe1ae0e4ad55fb431fdbc